src/lib/y2storage/devicegraph_sanitizer.rb

```diff
@@ -35,24 +37,19 @@ module Y2Storage
   #   sanitizer = DevicegraphSanitizer.new(devicegraph)
   #   new_devicegraph = sanitizer.sanitized_devicegraph
   class DevicegraphSanitizer
-    include Yast::I18n
-
     # @return [Devicegraph]
     attr_reader :devicegraph
 
     # Constructor
     #
     # @param devicegraph [Devicegraph] devicegraph to sanitize
     def initialize(devicegraph)
-      textdomain "storage"
```

Comment:
Are you sure this is no longer needed?
Aren't there still translated user messages in this file?
Won't those translations break when this `textdomain` is removed?

Not 100% sure, but IIRC we need that `textdomain` once for giving the translation extraction tools (`xgettext`) a hint what `.po` file to put the message to and also for the `_(...)` call to know what file to extract the message from.

Does this still work here without the `textdomain`?

Reply:
Yes, we still need it. But it was simply moved to a more internal class, which has now the translations, see `Error` class in that same file.

Reply:
At first glance it had appeared to me that this class still has user messages. But you are right: The messages are now all in different classes; in the same file, but in different classes.